cnxdb/migrations/20170912141914_derived-ruleset-trig.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+
+
+# Uncomment should_run if this is a repeat migration
+# def should_run(cursor):
+#     # TODO return True if migration should run
+
+
+def up(cursor):
+    cursor.execute("""CREATE OR REPLACE FUNCTION derived_book_ruleset()
+ RETURNS trigger
+ LANGUAGE plpgsql
+AS $function$
+BEGIN
+PERFORM * FROM module_files WHERE module_ident = NEW.module_ident and filename = 'ruleset.css';
+IF NOT FOUND THEN
+    INSERT INTO module_files (module_ident, fileid, filename)
+        SELECT NEW.module_ident, fileid, filename
+            FROM module_files
+            WHERE module_ident = NEW.parent AND filename = 'ruleset.css' ;
+END IF;
+END;
+$function$;
+
+create trigger duplicate_ruleset_for_derived AFTER INSERT ON modules FOR EACH ROW WHEN (NEW.portal_type = 'Collection' AND NEW.parent is not NULL and NEW.version = '1.1' and NEW.major_version = 1 and NEW.minor_version = 1) execute procedure derived_book_ruleset();
+""")
+
+
+def down(cursor):
+    cursor.execute("""drop trigger duplicate_ruleset_for_derived on modules;
+    drop function derived_book_ruleset""")
```

Comment:
The tests are saying there is a sql syntax error here. 🤔  Aren't the function parameters required to specify which function is to be dropped? I'm guessing `drop function derived_book_ruleset()`?